server/server.go

```diff
@@ -150,6 +150,22 @@ func (server *HTTPServer) getContentForRules(
 	return hitRules, totalRules, nil
 }
 
+// getContentForRules returns the hit rules from the report, as well as total count of all rules (skipped, ..)
+func (server *HTTPServer) getUserVoteForRules(
+	writer http.ResponseWriter,
+	feedbacks map[types.RuleID]types.UserVote,
+	rulesContent []types.RuleContentResponse,
+) []types.RuleContentResponse {
```

Comment:
this is interesting - the function seems to change the slice + return the same slice